docs/3.x.x/getting-started/quick-start-tutorial.md

```diff
@@ -0,0 +1,397 @@
+
+### The Category Content Type
+
+The Category Content Type will have a **String** field named `category` for categories assigned to restaurants, and a **Relation field** with a **Many to Many** relationship.
+
+Let's now add this second content type. Click the **+ Add Content Type** menu item.
+
+![Add Content Type Menu Item](../assets/quick-start-detailed/AddContentTypeMenuItem.png 'Add Content Type Menu Item')
+
+Now enter a name for this new **Content Type** (we will call this one `category`), and write a Description. We will write `Restaurant Categories` for the description. Then click the **Save** button.
+
+![Category Content Type Name and Description](../assets/quick-start-detailed/categoryNameAndDescription.png 'Category Content Type Name and Description')
+
+Now you are ready to add the content type fields. We will add a **String** field for the Name and a **Relation Field** for creating a **Many to Many** relation between the Category Content Type and Restaurant Content Type. Click on the **+ Add New Field** button.
+
+![Category Add New Field Button](../assets/quick-start-detailed/categoryAddNewField.png 'Category Add New Field Button')
+
+Next we will click on the **String** field.
+
+![Category add String Field](../assets/quick-start-detailed/categoryAddStringField.png 'Category add String Field')
+
+You should be under the **BASE SETTINGS** tab, in the **Name** field, type `Name`. This will be the name of the category.
+
+![Category String Field Name](../assets/quick-start-detailed/categoryStringNameField.png 'Category String Field Name')
+
+Now, click on the **ADVANCED SETTINGS** tab, check **Required field** and **Unique field**. This field is required for each category and is not optional. Also, we want each category to have a unique name, so we check the **Unique field** box. And then click the **Continue** button.
+
+![Category String Field Advanced Settings](../assets/quick-start-detailed/categoryStringAdvancedSettings.png 'Category String Field Advanced Settings')
+
+We are now ready to add the second field, the **Relation** field for the **Many to Many relation** setting between Categories and Restaurants. Click the **+ Add New Field** button.
+
+![Category Add New Field for Relation Field](../assets/quick-start-detailed/categoryAddClickNewFieldRelation.png 'Category Add New Field for Relation Field')
+
+From here, click the **Relation** field.
+
+![Category Click Relation Field](../assets/quick-start-detailed/clickRelationFieldCategory.png 'Category Click Relation Field')
+
+This will bring you to the **Add New Relation** screen. We will change two elements on this page for the **Relation** field.
+
+![Add new Relation Field screen](../assets/quick-start-detailed/addNewRelationScreen.png 'Add new Relation Field screen')
+
+The first element to change is **Permission (Users-Permissions)** to **Restaurant** (on the right).
```

Comment:
I would say: Click on Permission (Users-Permissions) and change it to Restaurant (on the right side).

Reply:
Done. Changed.